source/slang/slang-syntax.h

```diff
@@ -753,8 +753,8 @@ namespace Slang
 
         struct Iterator
         {
-            Element* m_cursor;
-            Element* m_end;
+            const Element* m_cursor;
+            const Element* m_end;
```

Comment:
I don't think that `FilteredMemberList` had specifically implied it was a read-only iterator over the collection, but I would also concede/agree that there is no reasonable use case that would require it to provide mutable access.